packages/interfaces/src/kobo/mapping/flood.ts

```diff
@@ -133,6 +134,7 @@ export const floodSpecificKeys = {
 } as const;
 
 export enum FloodSpecific {
```

Comment:
Isn't this simply `KeyOf[floodSpecificKeys]` ?

Reply:
This string enums is useful to to call the keys at other places in the code. I am going to refactor the code so the specificKeys use the enums for their keys

Reply:
Even if you construct if,with KeyOf, typescript should be smart enough to let you call individual keys, no?